### ISSUE TITLE
Bloom Blob Missing Specifier in `firebase@10.12`

### ISSUE THREAD

**Issue by chrisspiegl** [firebase/firebase-js-sdk#8319]
### Operating System

macOS 14.5

### Browser Version

irelevant

### Firebase SDK Version

10.12.0

### Firebase SDK Product:

Analytics, AppCheck, Auth, Firestore, Performance

### Describe your project's tooling

`Nuxt@3.12.1`
`nuxt-vuefire@1.0.2`
`firebase-admin@12.1.1`

### Describe the problem

I upgraded to the latest version of `firebase` with any of the versions above `10.12.0` and get the following errors when running my project:

```
⚙ [nuxt] warmup for @firebase/webchannel-wrapper/bloom-blob failed with: Missing "./bloom-blob" specifier in "@firebase/webchannel-wrapper" package
⚙ [warmup] tracking dependencies for @firebase/webchannel-wrapper/bloom-blob failed with: Missing "./bloom-blob" specifier in "@firebase/webchannel-wrapper" package
```

These do not happen with version `firebase@10.11` (tested `10.11.1`) and have not happened before.
When downgrading to `firebase@10.11` the issues are no longer present.

I hope this can be fixed and upgraded so we can safely upgrade to `v10.12`.

### Steps and code to reproduce issue

Install the described dependencies. Especially `firebase@10.12`.
Then run the build and the issue comes up in development (after the browser urls are shown).

**Comment by ehsannas** [comment]
Thanks for reporting this @chrisspiegl . I'll take a look.

**Comment by ehsannas** [comment]
@chrisspiegl I just found out that we have a typo in one of our package.json files. I've posted https://github.com/firebase/firebase-js-sdk/pull/8331 to fix the typo, but I'm not certain if that's the root cause of the error you're seeing. Could you manually fix this in your project and let me know if it helps?

**Comment by chrisspiegl** [comment]
@ehsannas thank you for looking into this. I tried looking how I may be able to check this typo but did not find a way to do this. It appears I would have to completely build the SDK myself, which I currently do not have the time to do / test / figure out. Sorry 😢.

**Comment by ehsannas** [comment]
I haven't been able to reproduce the error unfortunately. 
<img width="2540" alt="image" src="https://github.com/firebase/firebase-js-sdk/assets/3828215/1cda7deb-9667-45ed-85bd-dde2526fde6c">


My package.json is:

```
{
  "name": "nuxt-app",
  "private": true,
  "type": "module",
  "scripts": {
    "build": "nuxt build",
    "dev": "nuxt dev",
    "generate": "nuxt generate",
    "preview": "nuxt preview",
    "postinstall": "nuxt prepare"
  },
  "dependencies": {
    "firebase": "10.12.0",
    "firebase-admin": "12.1.1",
    "nuxt": "^3.12.2",
    "nuxt-vuefire": "1.0.2"
  }
}
```

Could you point me to a minimal reproduction repository?

**Comment by chrisspiegl** [comment]
@ehsannas Is the #8331 released in a version?
Maybe I can check with that and see if the issue is still present.

**Comment by ehsannas** [comment]
It'll be released soon after https://github.com/firebase/firebase-js-sdk/pull/8345 lands

**Comment by chrisspiegl** [comment]
@ehsannas I looked into this again and tried to figure out if #8331 has been released. To me, it looks like it's not been released. Just updating here since the bot made that comment.

**Comment by dlarocque** [comment]
> @ehsannas I looked into this again and tried to figure out if #8331 has been released. To me, it looks like it's not been released. Just updating here since the bot made that comment.

Hi @chrisspiegl, this change was released in `10.12.3`, but I accidentally excluded this change from the [release notes](https://firebase.google.com/support/release-notes/js#version_10123_-_july_3_2024). 

The change was added to the changelog in https://github.com/firebase/firebase-js-sdk/commit/52f8deb1dc07cd6acc323f36d8be158e927d17c6, and you can also see it listed in the changes in the [GitHub release](https://github.com/firebase/firebase-js-sdk/releases/tag/firebase%4010.12.3).

I'm sorry for any confusion- my bad!

**Comment by chrisspiegl** [comment]
I updated to version `10.12.4` in our code base and the errors from my initial issue are gone.

Thank you. This appears to be fixed.
